src/components/comparison/layers-panel/list-item-wrapper/list-item-wrapper.tsx

```diff
@@ -82,38 +85,53 @@ const OptionsIcon = styled.div`
   }
 `;
 
-export const ListItemWrapper = forwardRef(
-  (props: BaseMapsItemProps, ref: ForwardedRef<HTMLDivElement>) => {
-    const {
-      children,
-      id,
-      selected,
-      hasOptions,
-      expandState,
-      onOptionsClick,
-      onClick,
-      onExpandClick,
-    } = props;
-    const onExpandClickHandler = (e: SyntheticEvent) => {
-      e.stopPropagation();
-      onExpandClick && onExpandClick();
-    };
-    return (
-      <Container ref={ref} checked={selected} onClick={onClick}>
-        <ItemContentWrapper>{children}</ItemContentWrapper>
-        {hasOptions && onOptionsClick && (
+export const ListItemWrapper = ({
+  children,
+  id,
+  selected,
+  hasOptions,
+  isOptionsPanelOpen = false,
+  optionsContent,
+  expandState,
+  onOptionsClick,
+  onClickOutside,
+  onClick,
+  onExpandClick,
+}: BaseMapsItemProps) => {
+  const onExpandClickHandler = (e: SyntheticEvent) => {
+    e.stopPropagation();
+    onExpandClick && onExpandClick();
+  };
+
+  return (
+    <Container checked={selected} onClick={onClick}>
+      <ItemContentWrapper>{children}</ItemContentWrapper>
+      {hasOptions && onOptionsClick && optionsContent && (
```

Comment:
Can we have here `optionsContent` only and get rid of `hasOptions`

Reply:
Removed `hasOptions`